### PR TITLE
chore: fail fast if something errors prior to running the test fn

### PR DESCRIPTION
**Pull request by warber**
#### What this PR does / Why we need it:
This PR adjusts the assertions, so that the test is aborted as soon as an error occurs before executing the test case.

### CODE REVIEW

File: cmd/monaco/integrationtest/account/all_resources_integration_test.go [hunk 1]
```diff
@@ -42,7 +43,7 @@ func TestDeployAndDelete_AllResources(t *testing.T) {
 	cliDeployMZones.SetArgs([]string{"deploy", "testdata/all-resources/manifest-mzones.yaml"})
 
 	err := cliDeployMZones.Execute()
-	assert.NoError(t, err)
+	require.NoError(t, err)
```
Comment (jskelin) [inline cmd/monaco/integrationtest/account/all_resources_integration_test.go:46]: Will this work for local execution during debugging (where the version is not set to 2.x)?

**Review by jskelin — APPROVED** [review]

File: cmd/monaco/integrationtest/account/all_resources_integration_test.go (hunk 1 above)
Comment (warber) [reply cmd/monaco/integrationtest/account/all_resources_integration_test.go:46]: I don't understand how this relates to the version being set?

**Review by Laubi — APPROVED** [review]

File: cmd/monaco/integrationtest/account/all_resources_integration_test.go (hunk 1 above)
Comment (jskelin) [reply cmd/monaco/integrationtest/account/all_resources_integration_test.go:46]: Sorry, wrongly reported...

Comment (warber) [reply cmd/monaco/integrationtest/account/all_resources_integration_test.go:46]: no problem :)

**Pull request merged**